app/src/main/res/values-vi/strings.xml

```diff
@@ -73,15 +73,15 @@
 
     <string name="off_type">Hình thức nghỉ</string>
     <string name="annual_leave">Nghỉ phép thường niên (14.5 ngày)</string>
-    <string name="leave_for_marriage">Kết hôn (3.0 ngày)</string>
-    <string name="leave_for_child_marriage">Con kết hôn (1.0 ngày)</string>
-    <string name="funeral_leave">Nghỉ lễ tang (3.0 ngày)</string>
-    <string name="leave_for_care_of_sick_child">Con ốm (20.0 ngày)</string>
-    <string name="pregnancy_examination_leave">Nghỉ khám thai (5.0 ngày)</string>
-    <string name="sick_leave">Nghỉ đau ốm (60.0 ngày)</string>
-    <string name="miscarriage_leave">Nghỉ sẩy thai (50.0 ngày)</string>
-    <string name="maternity_leave">Nghỉ thai sản (180.0 ngày)</string>
-    <string name="wife_labor_leave">Vợ sinh con (14.0 ngày)</string>
+    <string name="leave_for_marriage">Kết hôn (%s days)</string>
+    <string name="leave_for_child_marriage">Con kết hôn (%s days)</string>
+    <string name="funeral_leave">Nghỉ lễ tang (%s days)</string>
```

Comment:
WARNING: [StringFormatCount] Inconsistent number of arguments in formatting string `funeral_leave`; found both 0 and 1
 When a formatted string takes arguments, it usually needs to reference the same arguments in all translations (or all arguments if there are no translations.

There are cases where this is not the case, so this issue is a warning rather than an error by default. However, this usually happens when a language is not translated or updated correctly.